src/main/java/Hex.java

```diff
@@ -1,10 +1,21 @@
+import java.util.ArrayList;
+import java.util.List;
+
 public class Hex {
     ResourceType type;
+    List<Hex> neighbors;
     int position;
 
-    public Hex(ResourceType type, int position) {
+
+    public Hex(ResourceType type) {
         this.type = type;
-        this.position = position;
+        this.neighbors = new ArrayList();
+//        this.position = position;
+    }
+
+
+    public void connect(){
+
```

Comment:
What is this for?

Reply:
to add neighbors, gonna be randomly assigned. Or makes more sense to randomly assign the resources to the tiles?